lib/events/api.go

```diff
@@ -63,10 +63,24 @@ const (
 	// the beginning
 	SessionByteOffset = "offset"
 
-	// Join & Leave events indicate when someone joins/leaves a session
-	SessionJoinEvent  = "session.join"
+	// SessionJoinEvent indicates that someone joined a session
+	SessionJoinEvent = "session.join"
+	// SessionLeaveEvent indicates that someone left a session
 	SessionLeaveEvent = "session.leave"
 
+	// UserLoginEvent indicates that a user logged into web UI or via tsh
+	UserLoginEvent = "user.login"
+	// LoginMethod is the event field indicating how the login was performed
+	LoginMethod = "method"
+	// LoginMethodLocal represents login via tsh login
+	LoginMethodLocal = "local"
```

Comment:
what's the difference between `userpass` and `local` from admins perspective and why do you have to distinguish two? I would imagine that only `local` should suffice.

Reply:
Yes, that makes sense, I'll change.